frontend/fingertips-frontend/generated-sources-extended/index.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  AgeData,
+  HealthDataForArea as HealthDataForAreaAPI,
+  HealthDataPoint as HealthDataPointAPI,
+  IndicatorSegment as IndicatorSegmentAPI,
+  IndicatorWithHealthDataForArea as IndicatorWithHealthDataForAreaAPI,
+  SexData,
+} from '../generated-sources/ft-api-client';
+
+// MUST EXPORT THE * FIRST FROM THE ORIGINAL SOURCES
+export * from '../generated-sources/ft-api-client';
+
+// now we can export and override the types that the UI would like to use a little differently
+export interface HealthDataPoint extends Omit<HealthDataPointAPI, 'value'> {
+  year?: number; // remove this later
+  periodLabel?: string; // useful for UI to populate this rather than regenerate over and over
+  sex?: SexData;
+  ageBand?: AgeData;
+  value?: number;
```

Comment:
Stupid question. We omit `value` for the original type - only to add it back again.